docs/hive-integration/kube-api-getting-started.md

```diff
@@ -0,0 +1,493 @@
+time="2021-06-28T21:42:20Z" level=info msg="the amount of nmStateConfigs included in the image is: 1" func="github.com/openshift/assisted-service/internal/controller/controllers.(*InfraEnvReconciler).ensureISO" file="/go/src/github.com/openshift/origin/internal/controller/controllers/infraenv_controller.go:288" go-id=654 infra_env=myinfraenv infra_env_namespace=demo-worker4 request_id=0d9c0da4-beab-4a41-85a3-4444728dd6f0
+```
+
+###### Inspect Assisted Service Cluster Events
+Get the full event list here:
+```bash
+curl -s -k $(kubectl -n demo-worker4 get agentclusterinstalls.extensions.hive.openshift.io test-agent-cluster-install -o=jsonpath="{.status.debugInfo.eventsURL}")  | jq "."
+```
+
+At this point, you should expect to see events such as:
+```json
+  {
+    "cluster_id": "02a89bb9-6141-4d14-a82e-42f254217502",
+    "event_time": "2021-06-28T21:01:31.938Z",
+    "message": "Registered cluster \"test-cluster\"",
+    "severity": "info"
+  },
+  {
+    "cluster_id": "02a89bb9-6141-4d14-a82e-42f254217502",
+    "event_time": "2021-06-28T21:01:31.941Z",
+    "message": "Successfully registered cluster test-cluster with id 02a89bb9-6141-4d14-a82e-42f254217502",
+    "severity": "info"
+  },
+  {
+    "cluster_id": "02a89bb9-6141-4d14-a82e-42f254217502",
+    "event_time": "2021-06-28T21:42:20.768Z",
+    "message": "Generated image (Image type is \"minimal-iso\", SSH public key is not set)",
+    "severity": "info"
+  },
+```
+
+
+### Utilize BareMetalHost
+
+* Examples below generated by [dev-scripts](https://github.com/openshift-metal3/dev-scripts/) and were modified manually.
+
+#### 1. Create BMC Secret
+    
+
+```yaml
+cat <<EOF | kubectl create -f -
+apiVersion: v1
+kind: Secret
+metadata:
+  name: ostest-extraworker-4-bmc-secret
+  namespace: demo-worker4
+type: Opaque
+data:
+  username: foo # Change me
+  password: bar # Change me
+EOF
+```
+#### 2. Create [bareMetalHost](baremetal-agent-controller.md#creating-baremetalhost-resources)
+```yaml
+cat <<EOF | kubectl create -f -
+apiVersion: metal3.io/v1alpha1
+kind: BareMetalHost
+metadata:
+  name: ostest-extraworker-4
+  namespace: demo-worker4
+  annotations:
+    inspect.metal3.io: disabled
+  labels:
+    infraenvs.agent-install.openshift.io: "myinfraenv" # must match the infraEnv name and exist in the same namespace
+spec:
+  online: true
+  automatedCleaningMode: disabled
+  bootMACAddress: 00:cd:16:5c:68:47  # Change to your mac address
+  bmc:
+    address: redfish-virtualmedia+http://192.168.111.1:8000/redfish/v1/Systems/dd8071d3-4ab5-47c1-aa1d-b2f1e9db0d0c # Should be change to match your machine
+    credentialsName: ostest-extraworker-4-bmc-secret  # Must match the bmc secret you have just created
+EOF
+```
+
+##### Result
+* Node (VM in this case), turned on.
```

Comment:
It may be confusing to talk about a VM here. I think it's better in general to assume that a user of BMH is using real bare metal, and talk about simulating bare metal only where it's relevant.

Reply:
Also it's better to call this a "host" and reserve "Node" for describing a system running a kubelet in a cluster. [terminology guidance](https://github.com/metal3-io/metal3-docs/blob/master/design/nodes-machines-and-hosts.md#data-model)